src/context.ts

```diff
@@ -126,6 +135,7 @@ export class FreshReqContext<State>
     this.info = info;
     this.params = params;
     this.config = config;
+    this.isPartial = url.searchParams.has(PARTIAL_SEARCH_PARAM);
```

Comment:
Given that this state can be derived from `url`, can be replaced with a `get` method instead.